samples/matter/lock/README.rst

```diff
@@ -82,12 +82,12 @@ Device Firmware Upgrade support
 
 You can configure the sample to use the secure bootloader for performing over-the-air Device Firmware Upgrade using Bluetooth® LE, using the following build flags during the build process:
 
-* ``-DOVERLAY_CONFIG=../common/config/overlay-dfu_support.conf``
```

Comment:
Why doesn't the doc recommend using BUILD_WITH_DFU option?

Reply:
Somehow it does below, but I'm also not sure why it was changed. It seems that the change was introduced in https://github.com/nrfconnect/sdk-nrf/pull/5157. @FrancescoSer do you happen to remember why it was done like this, as for me we should recommend easier option, but I might not have full view of a problem.

Reply:
The scope of PR 5157 was to separate the list of the flags that needed to be set for running the build from the example shown, as it uses cmd and not SES. There might have been an issue in this specific case.

Just to clarify, which are the correct flags required for building with DFU in this case?

Reply:
> Just to clarify, which are the correct flags required for building with DFU in this case? If there is an issue there we can fix that in an additional commit.

Actually both of them, but inserting `-- -DCONF_FILE=prj_dfu.conf -DPM_STATIC_YML_FILE="configuration/build-target/pm_static.yml` was not user friendly, so we added `-- -DBUILD_WITH_DFU=1` option that basically does the same under the hood.

Reply:
@FrancescoSer I think that -DBUILD_WITH_DFU=1 flag should be also possible to add in SES, so I modified description a little bit to use simplified form and single option, but also left reference to setting cmake options instruction. Please verify if you are ok with that.

Reply:
```suggestion
You can configure the sample to use the secure bootloader for performing over-the-air Device Firmware Upgrade using Bluetooth® LE, using the ``-DBUILD_WITH_DFU=1`` build flag during the build process:
```
Then we also don't need line 85...